lib/http/headers.rb

```diff
@@ -94,7 +94,18 @@ def to_a
     #
     # @return [String]
     def inspect
-      "#<#{self.class} #{to_h.inspect}>"
+      "#<#{self.class} #{pretty_print}>"
+    end
+
+    # Returns a printable representation of headers.
+    #
+    # @param [String] separator
+    # @return [String]
+    def pretty_print(separator = ", ")
+      to_h.map do |name, value|
+        value = value.join("; ") if value.respond_to?(:join)
+        "#{name}: #{value}"
+      end.join(separator)
```

Comment:
Hmm. Good point. I never heard of any limitations on amount of lines returned by inspect. In Object documentation it says that it's a human readable object representation.

Now I'm not sure what will be the best option here. But if `inspect` will include headers, I would like it to be multiline rather than single line.